### DISCUSSION TITLE
When developing package, how to add laravel passport as the dependency without duplicating laravel itself?

### DISCUSSION THREAD

**Discussion by ProteanCode** [laravel/framework#43482] [Q&A]
Hello there

I am creating my own auth scaffolding package using Laravel Passport as its dependency.
I am however confused about my package having a Laravel framework in its vendor directory.

What if I that version will ship a Laravel 9 in its own directory, but be composer-required by Laravel 8?

Am I doing something wrong? I do not see that any of native packages that ships with Laravel itself have vendor directory on their own. 

My package is in sibling directory to /app (/packages/xxx/composer.json), so I guess this is why I have that fat vendor directory shipped with it when I do composer update

**Comment by henzeb** [comment]
It's a package registered on packagist. https://packagist.org/packages/laravel/passport maybe use that?
